apps/wakunode2/app.nim

```diff
@@ -548,6 +549,11 @@ proc startRestServer(app: App, address: ValidIpAddress, port: Port, conf: WakuNo
     let relayCache = TopicCache.init(capacity=conf.restRelayCacheCapacity)
     installRelayApiHandlers(server.router, app.node, relayCache)
 
+  ## Filter REST API
+  if conf.filter:
+    let filterCache = rest_filter_api.MessageCache.init(capacity=30)
```

Comment:
Should this be configurable like for `restRelayCacheCapacity` above, or at least defined in a constant? (I saw there are other "magic numbers" used below this when starting RPC server, but still:)

Reply:
You're absolutely right about it. I will quick fix it, also I need to sign the commit that I had not set up earlier though